features/step_definitions/admin_steps.rb

```diff
@@ -254,29 +263,39 @@ def update_settings(settings)
 
 When (/^I make a translation of an admin post$/) do
   visit new_admin_post_path
-  fill_in("admin_post_title", :with => "Deutsch Ankuendigung")
-  fill_in("content", :with => "Deutsch Woerter")
+  fill_in("admin_post_title", with: "Deutsch Ankuendigung")
+  fill_in("content", with: "Deutsch Woerter")
   step(%{I select "Deutsch" from "Choose a language"})
-  fill_in("admin_post_translated_post_id", :with => AdminPost.find_by_title("Default Admin Post").id)
+  fill_in("admin_post_translated_post_id", with: AdminPost.find_by_title("Default Admin Post").id)
   click_button("Post")
 end
 
 Then (/^I should see a translated admin post$/) do
   step(%{I go to the admin-posts page})
   step(%{I should see "Default Admin Post"})
-    step(%{I should not see "Deutsch Ankuendigung"})
+  step(%{I should see "Translations: Deutsch"})
   step(%{I follow "Default Admin Post"})
-  step(%{I should see "Translations: Deutsch Deutsch Ankuendigung"})
-  step(%{I follow "Deutsch Ankuendigung"})
+  step(%{I should see "Deutsch" within "dd.translations"})
+  step(%{I follow "Deutsch"})
   step(%{I should see "Deutsch Woerter"})
 end
 
+Then (/^I should see a translated admin post with tags$/) do
+  step(%{I go to the admin-posts page})
+  step(%{I should see "Default Admin Post"})
+  step(%{I should see "Tags: quotes futurama"})
+  step(%{I should see "Translations: Deutsch"})
+  step(%{I follow "Default Admin Post"})
+  step(%{I should see "Deutsch" within "dd.translations"})
+  step(%{I should see "futurama" within "dd.tags"})
+end
+
 Then (/^I should not see a translated admin post$/) do
   step(%{I go to the admin-posts page})
   step(%{I should see "Default Admin Post"})
   step(%{I should see "Deutsch Ankuendigung"})
   step(%{I follow "Default Admin Post"})
-  step(%{I should not see "Translations: Deutsch Deutsch Ankuendigung"})
+  step(%{I should not see "Translations: Deutsch"})
```

Comment:
We no longer show the title when we link to translations, so we don't want to look for the title (Deutsch Ankuendigung).